src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -137,6 +230,51 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public class firstTest {
```

Comment:
Usually classes are written with camel case starting with the first character being upper case. Can you make these examples a bit more normal.
As they stand right now, the constructors look like methods at first glance and can cause some confusion.

Reply:
please fix name

Reply:
Done @romani